apollo-client/src/main/java/com/ctrip/framework/apollo/util/ConfigUtil.java

```diff
@@ -410,23 +412,38 @@ public boolean isPropertyNamesCacheEnabled() {
     return propertyNamesCacheEnabled;
   }
 
+  public boolean isPropertyFileCacheEnabled() {
+    return propertyFileCacheEnabled;
+  }
+
   private void initPropertyNamesCacheEnabled() {
-    String propertyName = ApolloClientSystemConsts.APOLLO_PROPERTY_NAMES_CACHE_ENABLE;
-    String propertyEnvName = ApolloClientSystemConsts.APOLLO_PROPERTY_NAMES_CACHE_ENABLE_ENVIRONMENT_VARIABLES;
+    propertyNamesCacheEnabled = getPropertyBoolean(ApolloClientSystemConsts.APOLLO_PROPERTY_NAMES_CACHE_ENABLE,
+            ApolloClientSystemConsts.APOLLO_PROPERTY_NAMES_CACHE_ENABLE_ENVIRONMENT_VARIABLES,
+            "false");
+  }
+
+  private void initPropertyFileCacheEnabled() {
+    propertyFileCacheEnabled = getPropertyBoolean(ApolloClientSystemConsts.APOLLO_FILE_CACHE_ENABLE,
+            ApolloClientSystemConsts.APOLLO_FILE_CACHE_ENABLE_ENVIRONMENT_VARIABLES,
+            "true");
+  }
+
+  private boolean getPropertyBoolean(String propertyName, String envName, String defaultVal) {
     String enablePropertyNamesCache = System.getProperty(propertyName);
     if (Strings.isNullOrEmpty(enablePropertyNamesCache)) {
-      enablePropertyNamesCache = System.getenv(propertyEnvName);
+      enablePropertyNamesCache = System.getenv(envName);
     }
     if (Strings.isNullOrEmpty(enablePropertyNamesCache)) {
-      enablePropertyNamesCache = Foundation.app().getProperty(propertyName, "false");
+      enablePropertyNamesCache = Foundation.app().getProperty(propertyName, defaultVal);
     }
     if (!Strings.isNullOrEmpty(enablePropertyNamesCache)) {
       try {
-        propertyNamesCacheEnabled = Boolean.parseBoolean(enablePropertyNamesCache);
+        return Boolean.parseBoolean(enablePropertyNamesCache);
       } catch (Throwable ex) {
         logger.warn("Config for {} is invalid: {}, set default value: false",
```

Comment:
The `false` below should be replaced with the actual default value
```suggestion
        logger.warn("Config for {} is invalid: {}, set default value: {}",
```

Reply:
fixed